packages/react-examples/src/charting/AreaChart/AreaChart.Multiple.Example.tsx

```diff
@@ -197,11 +197,6 @@ export class AreaChartMultipleExample extends React.Component<{}, IAreaChartBasi
             legendsOverflowText={'Overflow Items'}
             yAxisTickFormat={d3.format('$,')}
             legendProps={{
```

Comment:
Just to confirm, did you check out these examples to verify that they still work as before (even without the FocusZone)? You can compare with the PR deploy site of charting in master. http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/

Reply:
I don't see any behavior changes in the Charting components after the `FocusZone` removal.